api/HTMLScriptElement.json

```diff
@@ -169,10 +169,10 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "4"
+              "version_added": "6"
```

Comment:
Where did the original "4" come from?

Reply:
Looks like it came from the wiki migration.

Reply:
Thanks for checking!